src/openzaak/tests/utils/cmis.py

```diff
@@ -124,6 +124,15 @@ class APICMISTransactionTestCase(CMISMixin, APITransactionTestCase):
 
 
 def serialise_eio(eio, eio_url, **overrides):
-    serialised_eio = json.loads(serializers.serialize("json", [eio,]))[0]["fields"]
+    serialised_eio = json.loads(
+        serializers.serialize(
+            "json",
+            [
+                eio,
```

Comment:
I think removing the trailing comma should undo this formatting

```suggestion
                eio
```